pkg/authorization/maximal_permission_policy_authorizer.go

```diff
@@ -89,27 +85,27 @@ type MaximalPermissionPolicyAuthorizer struct {
 func (a *MaximalPermissionPolicyAuthorizer) Authorize(ctx context.Context, attr authorizer.Attributes) (authorizer.Decision, string, error) {
 	if IsDeepSubjectAccessReviewFrom(ctx, attr) {
 		// this is a deep SAR request, we have to skip the checks here and delegate to the subsequent authorizer.
-		return a.delegate.Authorize(ctx, attr)
+		return DelegateAuthorization("deep SAR request", a.delegate).Authorize(ctx, attr)
```

Comment:
okay, so this is a place that will give us more info in the audit logs. I like it.